app/controllers/user_training_controller.rb

```diff
@@ -6,39 +6,59 @@
 
 class UserTrainingController < ApplicationController
   def index
-    config = Talentlms::Config.first
-    lms = Talentlms::Facade.new
+    lms = Talentlms::Facade.new(current_user)
+    courses = current_user.required_training_courses
+    configs = [].tap do |result|
+      courses.each { |course| result << course.config }
+    end.uniq
```

Comment:
Is this the same as the following?
```suggestion
    configs = courses.flat_map { |c| c.config }.uniq
```

Reply:
yes, its been added to the PR. Thanks!